README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+Homework 01 
+=========
+
+### By Jacob Sims Speyer
+
+#### October 7, 2019
+
+
+
+My name is Jacob Sims Speyer, and I am a third-year Public Policy Major from [Newton, Massachusetts](https://en.wikipedia.org/wiki/Newton,_Massachusetts). I plan on concentrating in health policy, but this may changed. I was inspired to pursue this path after taking a health economics class with Professor David Meltzer last quarter. On campus, I am involved in Model UN and Hillel. In Model UN, I am an Under-Secretary-General for MUNUC (our high school conference), where I oversee the work of multiple committees. I feel _quite strongly_ that Model UN fosters **cooperation and teamwork in an academic setting**. I am also on the administrative board of UChicago Hillel, where I voice feedback on programmatic changes at the organization. A few other things you may not know about me:  
+
+-   I have a twin brother named Josh, who currently attends Middlebury College. Josh is currently studying abroad in China, but he plans to go into medicine. Here is a photo of the two of us: 
+ 
+![](https://imgur.com/eO40UbV.jpg)
+
+*A photo of Josh and myself in San Francisco* 
+
+    
+-   Both my mother and my late father (along with the rest of my family) are South African. However, my family can be traced back to a small town in Germany during the Crusades, and pre-inquisition Portugal  
+
+-   I look up to my late father as a source of inspiration. Before he passed away last year, he reminded me that:
+
+>"You should never give up, even when you get tired of not giving up"
+
+
```

Comment:
- I think this is a great job. 
- This includes Headers, Emphasis (italics or bold), Lists, Images, Links as the professor suggested.
- It also includes Quote, which is excellent.
- His comment is very detailed, so we can find his process and efforts.

----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       *    |              |            |
| **Coding strategy**         |     *     |              |            |
| **Presentation**            |         *  |              |            |
| **Achievement, creativity** |    *       |              |            |
| **Ease of access**          |      *     |              |            |